mentoring/public/js/questionnaire.js

```diff
@@ -27,7 +29,11 @@ function MessageView(element, mentoring) {
                 popupDOM.css('maxHeight', data.height);
             } else {
                 popupDOM.css('height', '');
+                popupDOM.css('maxHeight', '');
             }
+            // .tip-choice-group should always be the same height as the popup
+            // for scrolling to work properly.
+            popupDOM.find('.tip-choice-group').height(popupDOM.height());
```

Comment:
@mtyaka this fixes the issue I had, but can't it be achieved using css rather than setting height via javascript?